solutions/classes/input-reader.js

```diff
@@ -0,0 +1,38 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, "utf-8");
+let idx = 0;
+data = data.split("\n");
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+const Inputreader = class {
```

Comment:
[naming conventions ](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions) are not full filled with variable, function and class names.  like `Inputreader`  `Data`  `read_integers` `read_strings`  `typeofinput ` etc.,